go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.8
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
-	github.com/ava-labs/avalanchego v1.12.2-0.20250106102004-902377d447ba
+	github.com/ava-labs/avalanchego v1.12.2-0.20250106214205-8885a5c44c63
```

Comment:
need to update the avalanchego commit in `versions.sh` to the first 8 bytes of the commit hash